graphql/schema.graphql

```diff
@@ -251,6 +252,12 @@ type StatusCount {
   status: String!
   count: Int!
 }
+# Returns task counts grouped by status for a build variant
+type GroupedStatusCount {
```

Comment:
Would naming it GroupedTaskCountByStatus everywhere work? Or is that too long? 
It took me reading that comment to realize what a "StatusCount" is.

Reply:
+1 for at least GroupTaskStatusCount